pom.xml

```diff
@@ -939,6 +939,15 @@
                     </execution>
                 </executions>
             </plugin>
+            
+             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <version>2.7</version>
+                    <configuration>
+                        <encoding>UTF-8</encoding>
```

Comment:
maybe `<encoding>${project.build.sourceEncoding}</encoding>`?
It's weird that you had to define it. Documentation states that ${project.build.sourceEncoding} is default value for encoding in this plugin